test/ruby/test_module.rb

```diff
@@ -519,6 +519,16 @@ def test_include_with_no_args
     assert_raise(ArgumentError) { Module.new { include } }
   end
 
+  def test_include_before_initialize
+    m = Class.new(Module) do
+      def initialize(...)
+        include Enumerable
+        super
+      end
+    end.new
+    assert_equal(true, m < Enumerable)
```

Comment:
How about using `assert_operator`?
```suggestion
    assert_operator(m, :<, Enumerable)
```